core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1599,31 +1622,37 @@ class TestMetricsReporter extends MetricsReporter with Reconfigurable with Close
   }
 
   override def reconfigurableConfigs(): util.Set[String] = {
-    util.Set.of(PollingIntervalProp)
+    util.Set.of(PollingIntervalProp, ReplicationConfigs.NUM_REPLICA_FETCHERS_CONFIG)
```

Comment:
@FrankYang0529 Thanks for your help. Could you please consider creating an individual metrics reporter for the new test? Including too many test cases in `TestMetricsReporter` could make it harder to understand

Reply:
Add another `TestNumReplicaFetcherMetricsReporter` for it.